.dockerignore

```diff
@@ -0,0 +1,10 @@
+**/*.json
```

Comment:
This helps reduce the amount of unused data Docker ships to its build context for every `docker build` operation.